test/distributed/test_dynamo_distributed.py

```diff
@@ -399,6 +399,11 @@ def opt_fn(inputs):
         self.assertTrue(same(correct_outputs, opt_outputs))
         self.assertEqual(check_splits_compiler.compiler_called, 3)
 
+        # ensure compatibilty with dynamo explain
+        _, _, _, _, break_reasons, _ = torch._dynamo.explain(ddp_m, inputs)
+        self.assertEqual(len(break_reasons), 3)
```

Comment:
nit, this might be a bit saner: 
```suggestion
        explain_out = torch._dynamo.explain(ddp_m, inputs)
        break_reasons = explain_out[4]
        self.assertEqual(len(break_reasons), 3)
```